script/Makefile

```diff
@@ -33,6 +33,8 @@ LINT_DEADLINE := 5m
 # Used to push pre-relase artifacts
 STAGING_IMAGE_NAME := docker.io/camelk/camel-k
 
+STAGING_RUNTIME_REPO := https://repository.apache.org/content/repositories/orgapachecamel-1160
```

Comment:
It's could be better if we can pass the staging repo url as an environment variable.

Reply:
This allows to switch the main repo to a staging version before we officially release it, making travis building in the meantime. If you want to customize it locally you can still run e.g. `make STAGING_RUNTIME_REPO=https://another-repo test-integration`